.github/workflows/deployment.yml

```diff
@@ -43,27 +51,19 @@ jobs:
         twine check dist/*
         twine upload -r testpypi --username __token__ --password ${{ secrets.TEST_PYPI_API_TOKEN }} --skip-existing dist/*
     - name: Publish to PyPi
-      if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+      if: github.event_name == 'push' && github.ref == 'refs/heads/main' && steps.previous_tag.outputs.tag < env.NEW_VERSION
```

Comment:
Does `<` operation work properly with the version numbers? was it enough to just use `!=`?

Reply:
It is, I checked it yesterday. But I will test it once again today.

Reply:
I decided to use < because it may prevent an accidental decrease of the verions

Reply:
Checked once again it works correctly